tox-win.ini

```diff
@@ -0,0 +1,19 @@
+[tox]
+envlist = py34,py35,py36,py37,cover
+skip_missing_interpreters = True
+
+[testenv]
+usedevelop = True
+deps = pytest
+       pytest-xdist
+       pyreadline
+       cover: pytest-cov
+commands = pip install -e acme
+           pytest -n auto --pyargs acme
+           pytest -n auto --pyargs certbot
+
+[testenv:cover]
+usedevelop = True
```

Comment:
I'm not sure if we need this here. Won't this setting be inherited from `testenv`?

Reply:
Indeed, but not relevant anymore with `tox-win.ini` refactoring.